# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdebase-workspace/cmake-check-libxi.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./kcontrol/kxkb/CMakeLists.txt	2009-08-24 12:54:07.000000000 +0200
+++ ./kcontrol/kxkb/CMakeLists.txt.new	2009-08-24 12:52:48.000000000 +0200
@@ -54,7 +54,7 @@
 endif(USE_XKLAVIER STREQUAL ON)
 macro_log_feature(X11_Xkbfile_FOUND "xkbfile" "X11 KXB library for keyboard handling software" "http://www.x.org" FALSE "" "xkbfile is required to build kxkb.")
 
-SET(CMAKE_REQUIRED_LIBRARIES -lXi)
+SET(CMAKE_REQUIRED_LIBRARIES ${X11_Xinput_LIB})
 CHECK_FUNCTION_EXISTS(_XiGetDevicePresenceNotifyEvent XiGetDevicePresenceNotifyEvent_EXISTS)
 
 if( X11_Xinput_FOUND AND XiGetDevicePresenceNotifyEvent_EXISTS )
